### PR TITLE
Added Julian's locations data and improved map styling

### PR DESCRIPTION
**Pull request by MAMV3x3**
I just added Julian's locations to the `locations.json` file and also changed the styling of the markers in the map.

image

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -40,18 +40,18 @@ def create_marker(location, color):
 
 
 def create_map(members):
-    m = folium.Map(min_zoom=2)
+    map = folium.Map(min_zoom=2)
```
Comment (gibbonhug) [inline app/__init__.py:43]: I like use of more explicit variable name.

**Review by gibbonhug — COMMENTED** [review]
Thank you for adding my data! The map looks great

**Review by gibbonhug — APPROVED** [review]
Approving changes this time instead of simply commenting

**Review by Pradyuman7 — APPROVED** [review]
🥇

**Pull request merged**